proposals/lip-0020.md

```diff
@@ -35,11 +35,7 @@ If `H` is the height from which on the proposed protocol is used, then the block
 
 ### BlockIDs in JSON Objects
 
-In a JSON object representing a block, all properties that contain a blockID have to be represented as a hexadecimal string. The properties of a block that contain a blockID are the `previousBlock` and the `id` property.
-
-### BlockID Verification
-
-During the verification of a block, the blockID must be verified as correct. If the blockID verification fails, the block has to be rejected. If `H` is the height from which on the proposed protocol is used, then a block `B` with `B.height < H` must have a valid blockID according to the current format (such a verification may happen, for example, during synchronisation).
+In a JSON object representing a block, all properties that contain a blockID have to be represented as a hexadecimal string. The properties of a block that contain a blockID are the `previousBlock` and the `id` property, whereby only the `previousBlock` property is required.
```

Comment:
I would not even mention `id` as a property of a block JSON object. Maybe something like: 
```suggestion
In a JSON object representing a block, the property `previousBlock`, containing a blockID, has to be represented as a hexadecimal string.
```

Also, going through the LIP I noticed a possible typo:
```suggestion
In the proposed protocol, the blockID of a transaction B is computed as follows:
```

Finally, I would change the very last sentence in the backwards compatibility section:
```suggestion
...since the format of the `previousBlock` property change.
```

Reply:
all changes applied